src/test/python/deep_qa_test/layers/complex_concat_test.py

```diff
@@ -48,4 +48,4 @@ def test_call_handles_complex_combinations(self):
                 input_4_tensor + input_3_tensor,
                 input_3_tensor
                 ], axis=1)
-        numpy.testing.assert_almost_equal(concat_tensor, expected_tensor, decimal=4)
+        numpy.testing.assert_almost_equal(concat_tensor, expected_tensor, decimal=3)
```

Comment:
I'm pretty sure the reason is the division by the output of `numpy.random.rand()`, which could be really small.